ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory.tsx

```diff
@@ -59,6 +61,11 @@ function ViolationsByPolicyCategory() {
     const query = getRequestQueryStringForSearchFilter(queryFilter);
     const { data: alertGroups, loading, error } = useAlertGroups(query, 'CATEGORY');
 
+    const isOptionsChanged =
+        lifecycle !== defaultConfig.lifecycle ||
+        sortType !== defaultConfig.sortType ||
+        xor(hiddenSeverities, defaultConfig.hiddenSeverities).length !== 0;
```

Comment:
Use of `xor` is probably clever enough to warrant a code comment.

https://florian.github.io/xor-trick/

Reply:
Fair enough, I was hoping for a `Set.prototype.equals()` method but sadly it does not exist. I'll add a comment here.